lib/puppet/module_tool/skeleton/templates/generator/Gemfile

```diff
@@ -1,7 +1,14 @@
-source 'https://rubygems.org'
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
```

Comment:
This seems like it's a source of conflict between modules created for local/internal use and modules created for forge. I'm not sure we should just automatically set this.